notebooks/MAST/HSC/CASJOBS/.gitignore

```diff
@@ -0,0 +1,4 @@
+
+*.png
+.ipynb_checkpoints
+.idea
```

Comment:
was ipynb_checkpoints not already working?  It seems to be in the root `.gitignore`... Also you might but `.idea` there too since that's a pretty general thing to skip!